CHANGELOG.md

```diff
@@ -46,6 +46,9 @@ Main (unreleased)
 
 - Add three new stdlib functions to_base64, from_URLbase64 and to_URLbase64 (@ravishankar15)
 
+- Reduce the memory consumption of an Alloy instance by 20MB.
+  If Alloy is running certain otelcol components, this reduction will not apply. (@ptodev)
```

Comment:
nit: the changelog entry is a bit cryptic, maybe it could include the fact that it's a patch of the goccy/go-json pkg?

Reply:
No worries, I reworded it to say "Use a forked `github.com/goccy/go-json` module"